docs/manual/common/concepts/ComponentTechnologies.md

```diff
@@ -12,7 +12,7 @@ As a complete microservices platform, Lagom assembles a collection of technologi
 
     * Akka's [Split Brain Resolver](https://tech-hub.lightbend.com/docs/akka-commercial-addons/current/split-brain-resolver.html) handles network failures and system crashes.
 
-    * [Lightbend Application Monitoring](https://www.lightbend.com/products/monitoring) provides real time monitoring of system health, availability and performance.
+    * [Lightbend's Cinnamon product](https://developer.lightbend.com/docs/cinnamon/latest/home.html) provides real time monitoring of system health, availability and performance.
```

Comment:
Isn't the commercial name `Lightbend Telemetry`?

Reply:
I've seen both used, but Cinnamon is by far the most popular, least confusing and most memorable.

Reply:
@bantonsson, @alanngai do you have a preference here?

Reply:
I would go with **Lightbend Telemetry (Cinnamon)** to cover all the bases